.github/workflows/pr-builder.yml

```diff
@@ -196,7 +196,7 @@ jobs:
 
   build:
     name: 🚧 Build
-    needs: [ lint, typecheck ]
+    needs: [ typecheck ]
```

Comment:
any reason to remove lint from build prerequisites?

Reply:
As there are a number of changed files, lint fixing is not considered under this effort. Therefore lint is removed to get the build job passed.